src/overlays/actors/ovl_En_Part/z_en_part.c

```diff
@@ -28,16 +30,114 @@ const ActorInit En_Part_InitVars = {
     (ActorFunc)EnPart_Draw,
 };
 
-#endif
+void EnPart_Init(Actor* thisx, PlayState* play) {
+}
+
+void EnPart_Destroy(Actor* thisx, PlayState* play) {
+}
+
+void func_80865390(EnPart* this, PlayState* play) {
+    f32 randVar;
+
+    this->unk144 = 1;
```

Comment:
```suggestion
    this->actionFuncIndex = 1;
```
Renamed to match previous suggestion.